pkg/controller/petset/pet_set_test.go

```diff
@@ -263,3 +263,12 @@ func TestPetSetBlockingPetIsCleared(t *testing.T) {
 		t.Errorf("Unexpected blocking pet, err %v: %+v", err, p)
 	}
 }
+
+func TestPetSetReplicaCount(t *testing.T) {
```

Comment:
Can you eithe write a unittest that checkst status.Replicas, or add a TODO to this to do so? (ideally the former). Since it really doesn't matter what syncPetSet returnes, all we care about is what is stored in apiserver db.